CMakeLists.txt

```diff
@@ -575,6 +575,13 @@ if(WIN32)
   add_compile_options(/Zl) # omit default library name in .OBJ
 endif(WIN32)
 
+#--------------------------------
+# Definition directives
+#  - all clr specific compile definitions should be included in this file
+#  - all clr specific feature variable should also be added in this file
+#----------------------------------
+include(clrdefinitions.cmake)
```

Comment:
The idea here is to move the inclusion of `clrdefinitions.cmake` up here to create an area of the CMake file where these definitions are active, but we have not yet included the PAL. This space (with CLR feature defs, but no PAL defs) is used to build the standalone GC.

There aren't any subdirectories being added between the old and new positions of `clrdefinitions.cmake`, so I don't think this has any side effects.